immich/.env-dist

```diff
@@ -44,8 +44,13 @@ IMMICH_MTLS_AUTH=false
 # Wildcards are allowed and encouraged on a per-app basis:
 IMMICH_MTLS_AUTHORIZED_CERTS=*.clients.immich.example.com
 
-# Enter the path on the host where you want Immich to upload new images (e.g., `/mnt/photos`, `~/images`, `../pictures`), or enter a name to be used as a named Docker volume which will be created for you (e.g., photos).
-IMMICH_UPLOAD_LOCATION=photos
+# Set to "false" to have Immich upload images to a named Docker volume,
+# or to "true" to have Immich upload images to a bind mount directory on
+# the host.
+IMMICH_USE_EXTERNAL_VOLUME=false
+# If IMMICH_USE_EXTERNAL_VOLUME=true, set to the path on the host that you
+# want Immich to upload new images to (e.g., `/mnt/photos` or `~/images`).
+IMMICH_UPLOAD_LOCATION=
```

Comment:
If this variable is only used when IMMICH_USE_EXTERNAL_VOLUME is true, then the variable name should reference that. I propose IMMICH_EXTERNAL_VOLUME.

Reply:
Fixed in [494f1b8](https://github.com/EnigmaCurry/d.rymcg.tech/pull/271/commits/494f1b8dcd1d9cac97ee4d883baa1f0c2a2282e1)